extensions/amp-story/amp-story.md

```diff
@@ -338,6 +338,18 @@ A URL to the [story poster](#posters) in square format (1x1 aspect ratio).
 
 A URL to the [story poster](#posters) in landscape format (4x3 aspect ratio).
 
+##### live-story [optional]
+
+Enables the [live stories](#Live-stories) functionality.
```

Comment:
Here and across the document: live story (singular)

The documentation mostly uses the singular form, and it'd make more sense to publishers since the attribute is singular too: `live-story`